### PR TITLE
Port clarification question test to master branch

### PR DESCRIPTION
**Pull request by idavidmcdonald**
We also rename the feature file to be more human readable rather than object focussed.

### CODE REVIEW

File: features/supplier/supplier_applies_for_an_opportunity.feature [hunk 1]
```diff
@@ -269,3 +269,16 @@ Scenario: Supplier changes their answers before submission
...
+
+@opportunity-clarification-question
+Scenario: Supplier asks a clarification question
+  Given that supplier is on that framework
+  And that supplier has a service on the digital-specialists lot
+  And I have a live digital-specialists brief
+  And I go to that brief page
+  And I click 'Ask a question'
+  Then I am on 'Ask a question about ‘Tea drinker’' page
+  And I enter 'test string' in the 'clarification-question' field
```
Comment (TheDoubleK) [inline features/supplier/supplier_applies_for_an_opportunity.feature]: I thought we were gong with "How do I ask a question?" here.

"test string" 100% sucks.

File: features/supplier/supplier_applies_for_an_opportunity.feature [hunk 2]
```diff
@@ -269,3 +269,16 @@ Scenario: Supplier changes their answers before submission
       | Talk snobbishly about water quality | First nice to have evidence |
       | Sip quietly                         |                             |
       | Provide biscuits                    |  Only the finest            |
+
+@opportunity-clarification-question
+Scenario: Supplier asks a clarification question
+  Given that supplier is on that framework
+  And that supplier has a service on the digital-specialists lot
+  And I have a live digital-specialists brief
```
Comment (TheDoubleK) [inline features/supplier/supplier_applies_for_an_opportunity.feature]: `And I have a live digital-specialists brief` is in the Background so we probably don't need to duplicate it here.

Comment (idavidmcdonald) [inline features/supplier/supplier_applies_for_an_opportunity.feature]: Good spot

**Comment by TheDoubleK** [comment]
:+1: nice.

**Pull request merged**